approve.php

```diff
@@ -0,0 +1,91 @@
+<?php
+// Start session
+session_start();
+
+require_once('header.php');
+
+// Connect to the database
+$hostname = "localhost"; 
+$database = "fgp";
+$username = "fgp";
+$password = "fgp";
+
+$connection = mysqli_connect($hostname, $username, $password, $database);
+
+// Check if the connection was successful
+if (!$connection) {
+    die("Connection failed: " . mysqli_connect_error());
+}
+
+// Function to update status
+function update_status($id, $new_status) {
+    global $connection; // Access the global $connection variable
+    $query = "UPDATE dbPersons SET status = '$new_status' WHERE id = '$id'";
+    $result = mysqli_query($connection, $query);
+    if (!$result) {
+        die("Update failed: " . mysqli_error($connection));
+    }
+}
+
+// Check if the "Approve" or "Reject" button is clicked
+if (isset($_POST['approve'])) {
+    $id = $_POST['id']; // Family ID
+    update_status($id, 'Active'); // Set status to 'Active'
+} elseif (isset($_POST['reject'])) {
+    $id = $_POST['id']; // Family ID
+    update_status($id, 'Rejected'); // Set status to 'Rejected'
+}
+
+// Query to fetch all families from the database
+$query = "SELECT * FROM dbPersons WHERE type = 'family' AND status = 'pending' ORDER BY last_name";
+$result = mysqli_query($connection, $query);
+
+if (!$result) {
+    die("Database query failed.");
+}
+
+// Display families and their status
+?>
+
+<table style="margin: auto; border-collapse: collapse;">
+    <tr>
+        <th style="text-align: center; padding: 10px;">Family</th>
+        <th style="text-align: center; padding: 10px;">Status</th>
+        <th style="text-align: center; padding: 10px;">Action</th>
+    </tr>
+
+<?php
+while ($row = mysqli_fetch_assoc($result)) {
+    ?>
+    <tr>
+        <td style="text-align: center; padding: 10px;"><?php echo $row['last_name']; ?></td>
+        <td style="text-align: center; padding: 10px;">
+    <?php
+    
```

Comment:
Need to decide on consistent capitalization for "status" variable, mixed capitalization here, lines 33 and 36 also use capitalized values.

Reply:
Let's do uppercase.